Yatima/Compiler/FromLean.lean

```diff
@@ -419,15 +424,15 @@ mutual
     | .const x xls _, .const y yls _ => do
       let univs ← concatOrds <$> (List.zip xls yls).mapM (fun (x,y) => cmpLevel x y)
       if univs != .eq then return univs
-      match names.contains x, names.contains y with
-      | true, true => return .eq
-      | false, true => return .gt
-      | true, false => return .lt
-      | false, false => do
+      match names.find? x, names.find? y with
+      | some nx, some ny => return compare nx ny
```

Comment:
This is looking up from the list with the incremental progress our `cmpExpr` iterations have made, right?

Reply:
Yes